internal/utility.go

```diff
@@ -174,13 +175,14 @@ func stripPrefixName(path string) string {
 }
 
 // TODO : unit tests
+var patternLSN = "[0-9A-F]{24}"
+var regexpLSN = regexp.MustCompile(patternLSN)
+
 // Strips the backup WAL file name.
 func stripWalFileName(path string) string {
-	name := stripBackupName(path)
-	name = strings.SplitN(name, "_D_", 2)[0]
-
-	if strings.HasPrefix(name, backupNamePrefix) {
-		return name[len(backupNamePrefix):]
+	founded_lsn := regexpLSN.FindAllString(path, 2)
+	if len(founded_lsn) > 0 {
+		return founded_lsn[0]
 	}
 	return ""
```

Comment:
Can we return something line +inf, not -inf?
So that we will not delete unknown files...